app/models/doc_result.rb

```diff
@@ -24,5 +38,7 @@ class Entity < Grape::Entity
     expose :docTitle, documentation: { type: 'String', desc: 'Title or primary descriptor of the linked result.' }
     expose :docSummary, documentation: { type: 'String', desc: 'If available, first paragraph or descriptor of the linked document.' }
     expose :docScore, documentation: { type: 'Float', desc: "If calculated, the relevance of the document result to the search request; i.e. the 'matching score'" }
+    expose :latitude, documentation: {type: 'String',desc: "Returns the latitude associated with node"}
+    expose :longitude, documentation: {type: 'String',desc: "Returns the longitude associated with node"}
```

Comment:
Here, should it be "Float"? Or is it String because we want to preserve the # of significant digits? (the latter makes sense to me... just want to be sure)